src/SAWScript/Interpreter.hs

```diff
@@ -2695,6 +2695,54 @@ primitives = Map.fromList
 
     -- TODO: jvm_alloc_multiarray
 
+  , prim "jvm_modifies_field" "JVMValue -> String -> JVMSetup ()"
```

Comment:
Are there test cases which use these new functions?

Reply:
I'll add some test cases to demonstrate the new primitives.

Reply:
The new test in `intTests/test_jvm_modifies` demonstrates all the new primitives.